SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/SpatialStatics.h"
+#include "GeneralProjectSettings.h"
+#include "SpatialConstants.h"
+#include "SpatialGDKSettings.h"
+#include "Utils/ActorGroupManager.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialStatics);
+
+bool USpatialStatics::SpatialNetworkingEnabled()
+{
+	return GetDefault<UGeneralProjectSettings>()->bSpatialNetworking;
+}
+
+bool USpatialStatics::SpatialOffloadingEnabled()
+{
+	return SpatialNetworkingEnabled() && GetDefault<USpatialGDKSettings>()->bEnableOffloading;
+}
+
+bool USpatialStatics::ActorGroupAssociationMatchesWorkerType(const AActor* Actor)
+{
+	if (Actor == nullptr)
+	{
+		return false;
```

Comment:
Should this count as an error case?

Reply:
How do you think this should be handled?

Reply:
The same way as the other errors below (not sure what @Vatyx will suggest but I don't see much that can be done unless you introduce a custom return type or somehow the logger which is used for blueprints is special and shows the errors in a different way). A quick look at `GameplayStatics` shows that they also do `UE_LOG(LogScript, ...` and in general just return the null value of the return type (in this case `false`). I see things like `return World ? World->GetTimeSeconds() : 0.f` a lot.

Reply:
Lol just read through a fair few `GameplayStatics` examples. Actually a bunch of these just null check and return false. Saw one where they use a `check(World)`. Will see what Sahil says I guess, this error log should be included at least.

Reply:
Agree, I think a Warning most likely should be included.